src/System.Net.Sockets.Legacy/tests/FunctionalTests/DnsEndPointTest.cs

```diff
@@ -58,8 +60,8 @@ public void Socket_ReceiveFromDnsEndPoint_ArgumentException()
         {
             using (Socket sock = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp))
             {
-                sock.Bind(new IPEndPoint(IPAddress.Loopback, 8080));
-                EndPoint endpoint = new DnsEndPoint("localhost", 8080);
+                sock.Bind(new IPEndPoint(IPAddress.Loopback, TestPortBase + 4));
```

Comment:
> There is another method where you specify port 0 and the OS automatically binds to the next available ephemeral port. The problem is that the code will need to make a syscall to find the actual port (see the Security tests). 

I thought about changing the tests to use this style, but decided against it for now. If nobody objects, I'll leave things as-is for now and open an issue to track the conversion.